apps/Hangover/install-64

```diff
@@ -0,0 +1,305 @@
+#!/bin/bash
+
+version=9.20
```

Comment:
unrelated

Reply:
the version variable actually is used

Reply:
It is only used (incorrectly) in the copy/pasted script from `Wine (x64)` from pi-apps which. Hence why I wrote "unrelated"